ql/src/java/org/apache/hadoop/hive/ql/exec/repl/AckTask.java

```diff
@@ -43,6 +43,9 @@
   @Override
   public int execute() {
     try {
+      for( preAckTask task : work.getPreAckTasks() ){
```

Comment:
Format line:   for( preAckTask task : work.getPreAckTasks() ){
       => for (preAckTask task : work.getPreAckTasks()) { ?